store.js

```diff
@@ -378,6 +378,15 @@ Store.prototype = {
         log.info("Adding repo " + repo.fullName);
         this.add(repo, cb);
     }
+,   updateRepo:    function (fullName, data, cb) {
+        this.getRepo(fullName, function(err, doc) {
+            if (err) return cb(err);
+            for (var k in data) doc[k] = data[k];
```

Comment:
Maybe `Object.assing({}, doc, data)` (or, if you want side effects `Object.assing(doc, data)` )? That is, are you sure you want doc to be modified instead of some intermediate object?